govcd/system.go

```diff
@@ -51,6 +51,14 @@ func CreateOrg(vcdClient *VCDClient, name string, fullName string, description s
 		OrgSettings: settings,
 	}
 
+	// There is a bug in the settings of CanPublishCatalogs.
```

Comment:
In the light of recent finding with load balancers and `bool` fields having `omitempty` (https://github.com/vmware/go-vcloud-director/issues/223)

Can this bug be caused that when `settings.OrgGeneralSettings.UseServerBootSequence = false` it is actually not sent via API?

Probably this is not this PRs scope, but I see danger in these `bool` fields having `omitempty` now. If user had `CanPublishCatalogs=true`, but decided to _update_ it to `false` - we would never set it because the fields would be skipped unmarshaled XML representation.

<img width="672" alt="image" src="https://user-images.githubusercontent.com/15804230/62277694-57cd5000-b44f-11e9-9af9-372438d15c8d.png">

Reply:
I added an explicit test for updates using these fields both in govcd and Terraform

Reply:
Thanks for the test. Apparently proxied NSX API and vCD API behave differently. When the field is skipped vCD API inherently sets it to `false`, while the proxied NSX API leaves previous value. This works well in your case.